css/styles.css

```diff
@@ -33,15 +59,31 @@ nav {
   font-size: 50px;
 }
 
+.textdeco2 {
+  color: #1a2236;
+  margin-top: 19px;
+  margin-left: 16px;
+  font-weight: 400;
+  font-family: 'Playfair Display', serif;
+  font-size: 50px;
+}
```

Comment:
Let's remember to add more spacing to the design to match the required Figma template 👍 

|  Your design   |  Figma design   |
| -------------- | -------------- |
|  <img width="411" alt="Screenshot 2022-10-31 at 18 46 27" src="https://user-images.githubusercontent.com/66112760/199074591-df50a978-1b4f-4598-ac2f-f13cfd57788d.png">  |   <img width="463" alt="Screenshot 2022-10-31 at 18 46 54" src="https://user-images.githubusercontent.com/66112760/199074664-da160c1d-6d79-434a-b62d-b135dd2a2eaf.png">   |